web/assets/styles/themes/madras.css

```diff
@@ -0,0 +1,41 @@
+/**
+ * Theme by Kuba Wolanin / github:kubawolanin
+ *
+ * Used "Madras" CSS Pattern by Divya Manian
+ * from http://lea.verou.me/css3patterns/#madras
+ */
+:root {
+  --body-bg: transparent;
+  --body-color: #191716;
+  --primary-color: hsl(29, 100%, 50%);
+  --box-bg: rgba(255, 255, 255, .9);
+  --header-bg: var(--body-bg);
+  --widget-design-bg: hsl(0, 0%, 0%);
```

Comment:
Maybe you want this a bit lighter?
The text is difficult to read:
![image](https://user-images.githubusercontent.com/2004147/27194005-47f82004-5201-11e7-98ff-c2b7c2f47a42.png)